notebooks/signac_106_Ideal_Gas_Example.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>typo ca -&gt; can</p><p>I'm not sure what you mean by search operations. Maybe something like "The project index is a convenient way to see all the job data in one place"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>This is a good question. Indexing is deprecated and we'll need to rewrite our tutorial content to remove those discussions/feature demos in the near future.</p>